### PR TITLE
Support embedding /book page directly

### LINKED ISSUE #6529: [CAL-814] /book endpoint can't be directly embedded.
Hi guys, can someone help me?
I just want to open the link '/omgitsshark/book?type=7&amp;date=2023-01-17T13%3A30%3A00%2B01%3A00&amp;slug=30min&amp;count=12&amp;getsignature' in embed solution by calLink but recieve 404 page, if I visit link directly(<https://staging.cal.profi.io/omgitsshark/book?type=7&amp;date=2023-01-17T13%3A30%3A00%2B01%3A00&amp;slug=30min&amp;count=12&amp;getsignature>) - all is fine. Also I want to extend Cal init payload to pass user information when user authorized in our application, do you have documentation about this?

[Slack Message](https://calendso.slack.com/archives/C04ACKUDN30/p1673952690235209?thread_ts=1673952690.235209&cid=C04ACKUDN30)

<sub>[CAL-814](https://linear.app/calcom/issue/CAL-814/book-endpoint-cant-be-directly-embedded)</sub>

### PR DESCRIPTION
**Pull request by hariombalhara**
## What does this PR do?

Fixes #6529

**Environment**:  Production

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How should this be tested?

- [x] Simply go to the playground at http://localhost:3100/embed/?only=ns:third#cal-booking-place-third-iframe and see that  Booking Form is directly shown

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: packages/embeds/embed-core/index.html [hunk 1]
```diff
@@ -272,7 +272,7 @@ <h3><a href="?only=hideEventTypeDetails">Hide EventType Details Test</a></h3>
             "inline",
             {
               elementOrSelector: "#cal-booking-place-third .place",
-              calLink: "pro/30min",
+              calLink: "pro/book/embed?type=3&duration=30&date=2023-03-21T14%3A30%3A00%2B05%3A30&slug=30min&count=&embed=",
```
Comment (hariombalhara) [inline packages/embeds/embed-core/index.html:275]: Update this calLink to test /book page for user event,

File: packages/prisma/seed.ts [hunk 2]
```diff
@@ -604,6 +608,24 @@ async function main() {
...
     ]
   );
+  if (team) {
+    for (const eventType of team.eventTypes) {
+      await prisma.eventType.update({
+        where: {
+          id: eventType.id,
+        },
+        data: {
+          users: {
```
Comment (hariombalhara) [inline packages/prisma/seed.ts:618]: As users weren't set, no timeslots were selectable in the seeded team events. They work now.

**Review by hariombalhara — COMMENTED** [review]
Self revie done

**Comment by PeerRich** [comment]
just fyi we try to get rid of /book soon and merge it with the previous component (cc @JeroenReumkens)

**Comment by hariombalhara** [comment]
Yeah, @JeroenReumkens informed me already. I am not sure when that would be done but in the meantime 2 users weren't able to embed that page, so created a PR.

**Comment by PeerRich** [comment]
yeah good temporary fix, my comment was more of an FYI

**Comment by emrysal** [comment]
This is not a temporary fix though, when users embed the book page directly it's part of our stack forever - gotta keep supporting this. would like @JeroenReumkens's feedback as well as move this out of the /:user, /team/:slug endpoints to its own dedicated EP for future proofing.

**Comment by hariombalhara** [comment]
That's a very good point @emrysal. I think then we shouldn't support /book embedding then. 
Other booking endpoints would remain same, only this one is going away.

I think we should close this PR.

**Pull request closed without merging**